src/proctored-exam-settings/proctoredExamSettings.scss

```diff
@@ -3,3 +3,10 @@ legend {
     font-size: 1.0rem;
     font-weight: 700;
 }
+
+.form-check:focus-within {
```

Comment:
NIT: I notice some shifting around visually when you focus on the element and then lose focus, which is different than the other inputs, which stay in place. Is that something you could adjust easily?

Reply:
I had noticed this, and it has to do with the fact that it is a border (from what I can tell). If I switch `border` to `outline` the shifting doesn't happen.